envs/environment.yaml

```diff
@@ -2,19 +2,19 @@
 #
 # SPDX-License-Identifier: MIT
 
-name: pypsa-eur
+name: pypsa-eur-nov14
```

Comment:
Not sure about this here, or is it not intended? If we want to do this, I'd rather use the pypsa-eur version as a suffix.

Reply:
Ups. Let's keep pypsa-eur.